aspnetcore/toc.md

```diff
@@ -6,6 +6,7 @@
 ## [Middleware](fundamentals/middleware.md)
 ## [Working with Static Files](fundamentals/static-files.md)
 ## [Routing](fundamentals/routing.md)
+## [URL Rewriting](url-rewriting.md)
```

Comment:
I wonder if `xref:url-rewriting` would've failed validation there.

Let me know if I should be using the `xref:` links everywhere.